pkg/apply/apply.go

```diff
@@ -74,9 +74,49 @@ func ApplyObject(ctx context.Context, client k8sclient.Client, obj *uns.Unstruct
 			}
 
 			return errors.Wrapf(err, "could not update object %s", objDesc)
-		} else {
-			log.Print("update was successful")
 		}
+		log.Print("update was successful")
```

Comment:
you right, hope its fine now logic, since the if block always return
the else can be removed,
if you prefer i will revert to original

Reply:
bump

Reply:
bump